kolibri/core/tasks/storage.py

```diff
@@ -115,6 +116,17 @@ def __contains__(self, item):
                 is not None
             )
 
+    @staticmethod
+    def recreate_default_tables(engine):
+        """
+        Recreates the default tables for the job storage backend.
+        """
+        Base.metadata.drop_all(engine)
+        scheduledjobs_base = declarative_base()
+        scheduledjobs_table = Table("scheduledjobs", scheduledjobs_base.metadata)
+        scheduledjobs_table.drop(engine, checkfirst=True)
+        Base.metadata.create_all(engine)
+
     def recreate_tables(self):
```

Comment:
This method is now unused, let's remove it in favour of the class method above.

Reply:
Done